app/policies/endpoint_policy.rb

```diff
@@ -3,4 +3,8 @@
 class EndpointPolicy < AccountScopedPolicy
   class Scope < ::AccountScopedPolicy::Scope
   end
+
+  def test?
+    true
```

Comment:
This would allow any user to trigger a test event for any endpoint. I think this should be allowed only if both the user and the endpoint belong to the same account

Reply:
Actually the endpoint would be filtered in controller. If the user is not authorized to see the endpoint, it will blow up in `find_endpoint`

Reply:
It works in a slightly different way, if I hit `/endpoints/:id/test`, it calls `test?` to check for permissions inside `find_endpoint`

```
/home/aruzicka/projects/mcom/notifications-backend/app/policies/endpoint_policy.rb:9:in `test?'
/home/aruzicka/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/pundit-2.0.1/lib/pundit.rb:221:in `public_send'
/home/aruzicka/.rbenv/versions/2.6.0/lib/ruby/gems/2.6.0/gems/pundit-2.0.1/lib/pundit.rb:221:in `authorize'
/home/aruzicka/projects/mcom/notifications-backend/app/controllers/endpoints_controller.rb:45:in `find_endpoint'
```